src/Quaternion.jl

```diff
@@ -19,7 +19,7 @@ Quaternion(a::Vector) = Quaternion(0, a[1], a[2], a[3])
 convert(::Type{Quaternion{T}}, x::Real) where {T} =
     Quaternion(convert(T, x), convert(T, 0), convert(T, 0), convert(T, 0))
 convert(::Type{Quaternion{T}}, z::Complex) where {T} =
-    Quaternion(convert(T, real(z)), convert(T, imag(z)), convert(T, 0), convert(T, 0))
+    Quaternion(convert(T, real(z)), convert(T, Base.imag(z)), convert(T, 0), convert(T, 0))
```

Comment:
Does this resolves #29 and #62?

Reply:
Yes, #62 but not #29. But I'm hoping to merge #52 instead to fix this.